### PR TITLE
Protects the syndicate frequency from interception by telecomms modifications

### PR DESCRIPTION
**Pull request by Shadowlight213**
People assumed that you required NTSL to be able to do this. NOPE!

### CODE REVIEW

**Comment by MrStonedOne** [comment]
Now see, this, this right here, would have been a good use of the [s] tag

File: code/game/machinery/telecomms/machine_interactions.dm [hunk 1]
```diff
@@ -254,9 +254,12 @@
 				if(newfreq && canAccess(usr))
 					if(findtext(num2text(newfreq), "."))
 						newfreq *= 10 // shift the decimal one place
-					if(!(newfreq in freq_listening) && newfreq < 10000)
-						freq_listening.Add(newfreq)
-						temp = "<font color = #666633>-% New frequency filter assigned: \"[newfreq] GHz\" %-</font color>"
+					if(newfreq == 1213)
```
Comment (optimumtact) [inline code/game/machinery/telecomms/machine_interactions.dm]: no defines for radio channels?

**Comment by optimumtact** [comment]
:+1:

File: code/game/machinery/telecomms/machines/bus.dm [hunk 2]
```diff
@@ -26,7 +26,8 @@
 
 	if(is_freq_listening(signal))
 		if(change_frequency)
-			signal.frequency = change_frequency
+			if(!(signal.frequency == SYND_FREQ))
```
Comment (duncathan) [inline code/game/machinery/telecomms/machines/bus.dm]: .....why not just use `!=`???

**Comment by tkdrg** [comment]
The code is awful but it's not your fault. And this is a pretty critical exploit. I'll manual merge this to fix the style issue.

**Comment by KorPhaeron** [comment]
HELP: Ktccd/(Syndicate Assault Cyborg-345) (?) (PP) (VV) (SM) (FLW) (TP) (REJT): ok, so I got a few bugs I need to ask about since this really suuuuucks as syndiborg: Why can nobody on the ops team hear my syndichannel talk? I hear theirs.... And 2nd, why are all the UI stuff on things like air alarms and APC's just blank screens? 

I think you broke syndiborg radios

**Comment by tkdrg** [comment]
He didn't, we tested this in the thunderdome the following round and it worked fine. Most likely, his teammates just ignored him as usual.

**Comment by optimumtact** [comment]
UI stuff could be unrelated nanoui3 changes

**Comment by Shadowlight213** [comment]
@KorPhaeron not possible that it was me. Syndie channel doesn't use normal tcomms. Also, it is completely impossible for the pr to have done anything, as the pr doesn't even touch normal broadcasting. It only prevents frequency changing for the syndie comms and prevents adding a filter manually via multitool. The filter can still be added via var edit or code.

**Pull request closed without merging**